core/csvcollector.class.inc.php

```diff
@@ -226,7 +226,15 @@ public function Fetch()
 
 			$this->Configure($aCsvHeaderColumns);
 
-			$this->CheckSynchroColumns();
+			Utils::Log(LOG_DEBUG, "[".get_class($this)."] Columns [".var_export($this->aSynchroColumns, true)."]");
+			$aColumnsToIgnore = [];
+			foreach ($this->aFields as $sSynchroColumn => $aDefs) {
+				if (array_key_exists($sSynchroColumn, $this->aSynchroFieldsToDefaultValues)
+					|| in_array($sSynchroColumn, $this->aIgnoredSynchroFields)) {
+					$aColumnsToIgnore[] = $sSynchroColumn;
+				}
+			}
+			$this->CheckColumns($this->aSynchroColumns, $aColumnsToIgnore, 'csv file');
```

Comment:
Method CheckColumn will be overloaded in this class to include lines 229:236

Reply:
Done